Kconfig.zephyr

```diff
@@ -94,9 +94,11 @@ config USE_CODE_PARTITION
 	help
 	  When selected application will be linked into chosen code-partition.
 
+DT_CHOSEN_Z_CODE_PARTITION := zephyr,code-partition
```

Comment:
What sourcery is this? Is this standard Kconfig? And if so, why not use the `zephyr,code-partition` string directly in the call to `dt_chosen_to_reg_addr()`?

Reply:
I did not read the commit message :)
Thanks for the details